cmd/add.go

```diff
@@ -216,6 +231,12 @@ func init() {
 		"pretend to be the this user (only works if you started the server)")
 	addCmd.Flags().StringVar(&setMetadata, "metadata", "",
 		"key=val;key=val metadata to apply to all files in the set")
+	addCmd.Flags().StringVar(&setReason, "reason", "backup",
```

Comment:
Might be an idea to create a [custom handler](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Var) for this, that will only accept those three values; would make some of the later code simpler.

Edit: If this set an enum/int (0 = backup, 1 = archive, 2 = quarantine) then it'd make `setReviewAndRemovalDurations` nicer to write as you'd be able to reference a global array to get the defaults, possibly using a `setIfEmpty` func to simplify the setting.